packages/icons/tests/chromatic/Icons.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import * as IconLibrary from "@hopper-ui/icons";
+import { LI, UL, type ColorValue, type ResponsiveProp } from "@hopper-ui/styled-system";
+import type { Meta, StoryObj } from "@storybook/react";
+
+interface ListProps {
+    size?: ResponsiveProp<"sm" | "md" | "lg">;
+    color?: ResponsiveProp<ColorValue>;
+}
+
+const List = ({ size, color }: ListProps) => {
+    const listItems = IconLibrary.iconNames.map(name => {
+        const Component = IconLibrary[name];
+
+
+        return (
+            <LI key={name} display="block">
+                <Component size={size} color={color} />
+            </LI>);
+    });
+
+    return (
+        <UL display="flex" flexWrap="wrap" alignItems="center" gap="inline-md" margin="core_0" padding="core_0" style={{ listStyle: "none" }}>
```

Comment:
@alexasselin008 size properties use a `-[size]` nomenclature, with the exception of dimension core tokens, which seem to use `_[size]` is there a reason for this?

**Example**
`margin="core_0"`

Reply:
Styled system props allow `semantic tokens`, `core tokens` and `dataviz` tokens values. We strongly recommend using semantic tokens whenever possible, but we understand that sometimes you really need other types of tokens. In order to easily identify when someone use a token that is not semantic, we prefix core tokens with `core_` and dataviz tokens with `dataviz_`. This helps classify them in the intellissense as well.

So the format is always the name with `-` but the prefix with `_`.